amundsen_application/static/css/_avatars.scss

```diff
@@ -0,0 +1,7 @@
+.sb-avatar > div {
+  border: 1px solid white;
+}
+
+.sb-avatar > img {
```

Comment:
just took this from the AvatarLabel component, might want to scope this styling down to just avatars inside of component-labels since this might lead to margin issues if we want sb-avatars with images in other elements